### ISSUE TITLE
model training in sagemaker

### ISSUE THREAD

**Issue by SSwethaSel0609** [k2-fsa/icefall#1383]
what are the steps to train the  icefall asr model in aws sagemaker

**Comment by csukuangfj** [comment]
Is aws sagemaker similar to google colab?

**Comment by SSwethaSel0609** [comment]
No, Amazon SageMaker is a cloud machine-learning platform that enables developers to create, train, and deploy machine-learning models in the cloud.
It makes it convenient to train models without worrying about compute(disk space, gpu unavailability etc.) 

It can also add TTV(Time To Value) as it reduces training time.

It can reduce cost for us

**Comment by csukuangfj** [comment]
Can you use it like a virtual machine? Does it support running docker? We provide docker images for icefall.

**Comment by SSwethaSel0609** [comment]
yes we need to create a docker image.. but I have docker image.. I don't know how to start

**Comment by csukuangfj** [comment]
Can you use ssh to login into the machines provided by ` aws sagemaker`?
